Dockerfile

```
@@ -1,9 +1,10 @@
-FROM golang:1.16.5-buster AS build-env
+FROM golang:1.16.5-alpine3.14 AS build-env
 
 # Install minimum necessary dependencies,
-ENV PACKAGES make git gcc
-RUN apt-get update -y
-RUN apt-get install -y $PACKAGES
+RUN set -eux; apk add --no-cache ca-certificates build-base;
+
+RUN apk add git
+RUN apk add linux-headers
```

```suggestion
RUN apk add libusb-dev linux-headers
```

Ah. As I know, the `libusb-dev` need to be installed, because we always enable the `ledger` tag in the Makefile.

---

It seems to be build without `livusb-dev`. Is this cause problems when the actual panacea run?

---

I guess so. I haven't tested it by myself, but referred to https://github.com/CosmWasm/wasmd/blob/9a17505cf8b637942cad78e294aa1377d7f7b2f6/Dockerfile#L11

---

Aha, I got it. Thank you!